### PR TITLE
Add a macrobenchmark for laying out text in a list

### PR DESCRIPTION
**Pull request by dnfield**
Adds a benchmark from https://github.com/flutter/flutter/issues/94419

### CODE REVIEW

File: dev/devicelab/lib/tasks/perf_tests.dart [hunk 1]
```diff
@@ -418,6 +418,13 @@ TaskFunction createsMultiWidgetConstructPerfE2ETest() {
   ).run;
 }
 
+TaskFunction createListTextLayoutPerfE2ETest() {
+  return PerfTest.e2e(
```
Comment (zanderso) [inline dev/devicelab/lib/tasks/perf_tests.dart:422]: `PerfTest.e2e` accepts an `enableImpeller` flag if you want to eagerly add an Impeller version: https://github.com/flutter/flutter/blob/master/dev/devicelab/lib/tasks/perf_tests.dart#L874

**Review by zanderso — APPROVED** [review]
lgtm w/ optional addition.

File: dev/devicelab/lib/tasks/perf_tests.dart (hunk 1 above)
Comment (dnfield) [reply dev/devicelab/lib/tasks/perf_tests.dart:422]: Done. This also mad eme realize one of the files was named `.dart.dart`... hah.

**Review by zanderso — COMMENTED** [review]
The impeller version will also need to go in the owners file.

**Review by goderbauer — APPROVED** [review]
LGTM

**Pull request merged**